oak/server/Cargo.lock

```diff
@@ -1126,15 +1340,27 @@ dependencies = [
 
 [[package]]
 name = "sha2"
-version = "0.9.0"
+version = "0.8.2"
```

Comment:
Why was this downgraded? I think the API is also different, I remember actually upgrading it to the new API at some point.

Reply:
Upgraded back
Downgraded version was needed to rust [ed25519](https://docs.rs/ed25519-dalek/0.9.1/ed25519_dalek/), but it's not used anymore